### PR TITLE
 adding the font awesome to Vue and installing dependencies

### PR DESCRIPTION
**Pull request by MouaadDilmi**
finishing the awesome front vue dependencies

### CODE REVIEW

**Review by gabrielheinrich — DISMISSED** [review]
Can you please remove the two package-lock.json files and try to use the faCoffee Icon in one of the components (e.g. Home.vue)

File: README.md [hunk 1]
```diff
@@ -22,16 +20,43 @@ Use: authentication, hosting and storage
 
 The application runs a backend server on fire stack for the data storage and authentication. It is started with the emulators. The frontend is implemented via vue and has to be stgarted separately.
 
-start:emulators:
+### start emulators
 
 ```bash
-firebase emulators:start
+yarn start
```
Comment (DorianWilhelm) [inline README.md]: the script to start the emulators is 'yarn start:emulators'

File: README.md [hunk 2]
```diff
@@ -22,16 +20,43 @@ Use: authentication, hosting and storage
...
 ```
 
-start:frontend:
+### start frontend
 
 ```bash
-cd frontend && yarn serve
+yarn serve
```
Comment (DorianWilhelm) [inline README.md]: the script to start the frontend is 'yarn start:frontend'

File: README.md [hunk 3]
```diff
@@ -22,16 +20,43 @@ Use: authentication, hosting and storage
...
+library.add(faCircle, faCheck);
+```
+
+use them as follows in kebab case
+
+```javascript
+// circle icon
+<font-awesome-icon icon="circle" />
+// check icon
+<font-awesome-icon icon="check" />
```
Comment (DorianWilhelm) [inline README.md:59]: Why are these instructions needed?

**Review by DorianWilhelm — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (gabrielheinrich) [reply README.md:59]: Just as a reference for everyone to know how to add font awesome extra icons in components

Comment (DorianWilhelm) [reply README.md:59]: got it

**Review by DorianWilhelm — DISMISSED** [review]

**Review by DorianWilhelm — APPROVED** [review]

**Pull request merged**